### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by Miciah**

### CODE REVIEW

File: rest_api/kubernetes_v1.adoc [hunk 1]
```diff
@@ -7094,7 +7094,7 @@ Deleting a persistent volume removes the cluster's record of the volume, and may
 === v1.Capability
 === v1.Pod
 :hardbreaks:
-A pod corresponds to a group of containers running together on the same machine. All containers in a pod share an IP address, and may have access to shared volumes and local fileystem. Like individual application containers, pods are considered to be relatively ephemeral rather than durable entities. Pods are scheduled to nodes and remain there until termination (according to restart policy) or deletion. When a node dies, the pods scheduled to that node are deleted. Specific pods are never rescheduled to new nodes; instead, they must be replaced by a component like the replication controller.
```
Comment (Miciah) [inline rest_api/kubernetes_v1.adoc]: The line is too long for GitHub to highlight the changed word, but "filesystem" was misspelled as "fileystem".

File: rest_api/kubernetes_v1.adoc [hunk 2]
```diff
@@ -7949,7 +7949,7 @@ Persistent Volume Claims (PVC) represent a request to use a persistent volume (P
 
 === v1.Service
 :hardbreaks:
-A Service is an abstraction which defines a logical set of Pods and a policy by which to access them - sometimes called a micro-service. The set of Pods targeted by a Service is (usually) determined by a Label selector. Services broadly fall into two types - those that load balance a set of Pods and hide which Pod a client talks to (clusterIP set to an IP address), and those where clients want to talk to the individual member pods directly (clusterIP set to 'None', also known as 'headless'services). The cluster IP of a service is exposed as an environment variable in each pod in the same namespace.
```
Comment (Miciah) [inline rest_api/kubernetes_v1.adoc]: There was a space missing between "'headless'" and "services".

File: rest_api/openshift_v1.adoc [hunk 3]
```diff
@@ -8595,7 +8595,7 @@ Upon log in, every user of the system receives a User and Identity resource. Adm
 
 === v1.NetNamespaceList
 :hardbreaks:
-NetNamespaceList represents a list of NetNamespace objects. NetNamespace catpures information about a segregated network namespace for an entire cluster. When a group of pods, or a project, or a group of projects get a NetNamespace assigned then the openshift-sdn's multitenant plugin ensures network layer isolation of traffic from other NetNamespaces.
```
Comment (adellape) [inline rest_api/openshift_v1.adoc]: IDK, I kinda like "cat purrs" better...

**Comment by adellape** [comment]
:bow:

**Comment by Miciah** [comment]
Thanks!

**Pull request merged**